Tests/Invoke-DockerPull.Tests.ps1

```diff
@@ -25,28 +28,28 @@ Describe 'Pull docker images' {
     Context 'Docker pulls docker images' {
 
         It 'pulls public docker image by image name only' {
-            Invoke-DockerPull -ImageName 'mcr.microsoft.com/windows/servercore/iis'
+            Invoke-DockerPull -Registry 'mcr.microsoft.com' -ImageName 'windows/servercore/iis'
```

Comment:
We need to fix this test, so the code fits what the `It` says. 😄 
I will make some commits.

Reply:
Fixed.